tests/authentication.test.js

```diff
@@ -136,4 +135,102 @@ describe('Login', () => {
         done();
       });
   });
+  // Email
+  it('reset correct password email', (done) => {
+    const user = {
+      email: 'ugizwenayodiny@gmail.com'
+    };
+    chai
+      .request(server)
+      .post('/api/auth/user/1')
+      .send(user)
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.message.should.be.equal('The email has been sent successfully');
+        done();
+      });
+  });
+  it('reset wrong password email', (done) => {
+    const user = {
+      email: 'ugizwenayodiny1@gmail.com'
+    };
+    chai
+      .request(server)
+      .post('/api/auth/user/1')
+      .send(user)
+      .end((err, res) => {
+        res.should.have.status(403);
+        res.body.error.should.be.equal('User does not exist');
+        done();
+      });
+  });
+  it('update the password', (done) => {
+    const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiZmlyc3ROYW1lIjoiRGl2aW5lIiwibGFzdE5hbWUiOiJ1Z2l6d2VuYXlvIiwiZW1haWwiOiJ1Z2l6d2VuYXlvZGlueUBnbWFpbC5jb20iLCJnZW5kZXIiOiJGZW1hbGUiLCJhZGRyZXNzIjoia2FtZW1iZSIsInJvbGUiOiJ1c2VyIiwiaXNWZXJpZmllZCI6ZmFsc2UsImNyZWF0ZWRBdCI6IjIwMjAtMDMtMTFUMDg6NTY6MTMuMDU3WiIsInVwZGF0ZWRBdCI6IjIwMjAtMDMtMTFUMDg6NTY6MTMuMDU3WiIsImlhdCI6MTU4MzkxNzEwMywiZXhwIjoxNTg0MDAzNTAzfQ.bwwiL9PwiEUtGxmmgCIAChMTph2-ZeoOfkuJi4fIpqk';
+    const pass = {
+      password: 'ugizwe'
+    };
+    chai
+      .request(server)
+      .post(`/api/auth/resetpassword/1/${token}`)
+      .send(pass)
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.message.should.be.equal('The password has been changed');
```

Comment:
same thing for the message, you can do this `res.body.message.should.be.equal(messages.passwordChanged)`